core/token_auth.go

```diff
@@ -7,7 +7,9 @@ import (
 
 	"github.com/pborman/uuid"
 
+	"github.com/starkandwayne/goutils/log"
```

Comment:
Minor formatting nit; since goutils is outside of SHIELD, move this import up to be next to pborman up there, not down with the "internal" SHIELD deps down here.

Reply:
Even better, use the correct log library.

Reply:
nvm, this is server side. Yeah, I can do that. But I think this might be a goimports thing. I'll see.